pkgs/package-overrides.nix

```diff
@@ -287,6 +287,20 @@ in
       else
         throw "php.extensions.redis3 requires PHP version < 8.0.";
 
+    tidy = prev.extensions.iconv.overrideAttrs (attrs: {
+      patches =
+        let
+          upstreamPatches =
+            attrs.patches or [];
+
+          ourPatches =
+            lib.optionals (prev.php.version == "5.6") [
```

Comment:
Would not the version have three components?

Reply:
I haven't build it for >= 5.6.

Reply:
The patch is only needed for PHP 5.6. Check the tests.

Reply:
Yeah, but the version would be 5.6.40, which would return false with `==`.